go/tasks/plugins/k8s/sagemaker/constants.go

```diff
@@ -0,0 +1,41 @@
+package sagemaker
+
+import commonv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/common"
+
+const (
+	pluginID          = "aws_sagemaker_hpo"
+	sagemakerTaskType = "aws_sagemaker_hpo"
+)
+
+
+const (
+	BayesianSageMakerAPIHyperParameterTuningJobStrategyType commonv1.HyperParameterTuningJobStrategyType = "Bayesian"
+	RandomSageMakerAPIHyperParameterTuningJobStrategyType commonv1.HyperParameterTuningJobStrategyType = "Random"
+)
+
+const (
```

Comment:
should these be enumer?

Reply:
if so should they be in the protobuf?

Reply:
This is a mapping from our protobuf type to sagemaker's API type tho?

Reply:
look at the previous comment